### DISCUSSION TITLE
Postgres function response schemas

### DISCUSSION THREAD

**Discussion by amacfie-tc** [PostgREST/postgrest#2729] [Q&A] [answered]
Right now the endpoints that are generated for Postgres functions have only a minimal specification of the response(s) in the OpenAPI file: `"responses":{"200":{"description":"OK"}}}}`. Is there a way to get the return type of the function represented in the OpenAPI schema? If not, is this a feature planned for the future?

**Comment by wolfgangwalther** [comment, marked as answer]
A quick [search through our issues](https://github.com/PostgREST/postgrest/issues?q=is%3Aissue+is%3Aopen+label%3AOpenApi) returns as the 2nd match: https://github.com/PostgREST/postgrest/issues/2180
